### ISSUE TITLE
a11y: Color Contrast - Home

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13675]
Insufficient color contrast for standard text

## Actual result

The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.

## Instances

"Foreground color: #3D98D3
Background color: #E2F0F8
Contrast ratio:  2.72:1
Main Content: 
- What's new in 2.9

Foreground color: #3D98D3
Background color: #FFFFFF
Contrast ratio: 3.16:1
Main Content: 
- Preferences
- Manage

Foreground color: #3D98D3
Background color: #E2F0F8
Contrast ratio: 2.72:1 
Main Content: 
- 2 

Foreground color: #FFFFFF
Background color: #3D98D3
Contrast ratio:  3.16:1
Main Content: 
- Import Existing
- Create

Foreground color: #BEC1D2
Background color: #FFFFFF
Contrast ratio: 1.78:1 
Main Content: 
- Filter

Foreground color: #5D995D
Background color: #FFFFFF
Contrast ratio: 3.39:1
Main Content: 
- Active

Foreground color: #B6B6C2
Background color: #F4F5FA
Contrast ratio: 1.81:1 
Main Content: 
- Distro (Hover and selected state)
- Architecture (Hover and selected state)
- K3s (Hover and selected state)
- RKE2 (Hover and selected state)
- amd64 (Hover and selected state) 

Foreground color: #B6B6C2
Background color: #FFFFFF
Contrast ratio: 2.00:1 
Main Content: 
- Distro (By default state)
- Architecture (By default state)
- K3s (By default state)
- RKE2 (By default state)
- amd64 (By default state) 

Foreground color: #3D98D3
Background color: #FFFFFF
Contrast ratio:  3.16:1
Main Content: 
- Docs
- Forums 
- Slack
-  File an Issue 
- Get Started 
and so on. 

## Expected Result

The contrast ratio between foreground text and background for the standard text should be sufficient.

## Recommended action

On all pages, ensure that each foreground/background color combination presents a contrast ratio of at least 4.5:1 for standard text.

**Comment by rak-phillip** [comment]
Issue 66 fails VPAT:

> "Actual Result: The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.
> 
> Instances:
> Foreground color: #BEC1D2
> Background color: #FFFFFF
> Contrast ratio: 1.78:1 
> Main Content: 
> - Filter
> 
> Recommendation: On all pages, ensure that each foreground/background color combination presents a contrast ratio of at least 4.5:1 for standard text."

Issue 67 fails VPAT:

> "Actual Result:
> The issue is still not fixed 
> Additionally 
> The focus indicator has insufficient contrast with background/foreground color combinations, making it difficult for users with low vision and color blindness to identify the focus.
> 
> Instances:
> - Cluster( selected control focus indicator fails)
>
> Foreground: #3259A8
> Background: #496192
> Contrast ratio: 1.1:1
> 
> Recommendation:
> Ensure that the color contrast of focus indicator for all user interface controls is 3:1 with the foreground and background color.
>
> "

Issue 68 - Removed

Issue 73 fails VPAT. No notes. Original comment:

> Some form control border have insufficient contrast with their background, making it difficult for people with low vision and color blindness to distinguish them.
